gpustack/policies/candidate_selectors/ascend_mindie_resource_fit_selector.py

```diff
@@ -203,7 +203,7 @@ async def _estimate_usage(self) -> RequestEstimateUsage:  # noqa: C901
         n_tokens = self._serving_params.max_seq_len
         n_layers = self._model_params.num_hidden_layers
         n_heads = 1
-        d_head = self._model_params.head_dim
+        d_head = self._model_params.head_dim or 0
```

Comment:
Seems to be fixed in https://github.com/gpustack/gpustack/pull/3218, but I don't see a PR conflict. Can you check?